Assets/Mirror/Editor/NetworkBehaviourInspector.cs

```diff
@@ -73,6 +77,9 @@ public override void OnInspectorGUI()
         /// </summary>
         protected void DrawDefaultSyncLists()
         {
+            // Need this check incase OnEnable returns early
+            if (syncListDrawer == null) { return; }
```

Comment:
is this related to the issue or does it fix another issue? if yes, please submit in a separate PR :)

Reply:
This is related, If the new code in `OnEnable` returns early `syncListDrawer` will be null.